libs/paths/Ppath.mli

```diff
@@ -51,6 +51,15 @@ val from_segments : string list -> (t, string) result
 (* Convert back to a system path. *)
 val to_fpath : root:Fpath.t -> t -> Fpath.t
 
+(* Convert from a system path to a normalized project path.
```

Comment:
Why adding those functions? I've changed Ppath.mli to be minimalistic and to not offer many way
to build ppath on purpose. A ppath should be part of a project, hence the in_project function builder above.
For your of_fpath, what will be the resulting project path?